src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -2121,6 +2288,7 @@ private void pushWaitingPeopleRecord() {
     // MUST BE THREAD SAFE. Called from crazy places. mPersistentIdentity may not exist
     // when this is called (from its crazy thread)
     private void sendAllPeopleRecords(JSONArray records) {
+        if (hasOptedOutTracking()) return;
```

Comment:
same as above, idk if `deleteUser` will work as expected due to this change.

Reply:
yup, it will!